xskillscore/core/probabilistic.py

```diff
@@ -73,12 +65,10 @@ def xr_crps_gaussian(observations, mu, sig, dim=None, weights=None):
             return res.mean(dim)
 
 
-def xr_crps_quadrature(
+def crps_quadrature(
```

Comment:
Useful to rename

Reply:
Yes I just imported properscoring with it's name so it's clear where we're using properscoring.